Index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>benjaminSemah.com</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="logo">
+            <a class="myLogo" href="#">My Logo</a>
+        </div>
+        <div class="navIcon">
+            <a href="#"><img src="assets/navIcon.svg" alt="navigation icon"></a>
+        </div>
+    </header>
+    <img id="whiteEllipse" src="/assets/ellipseHeader.svg" alt="ellipse">      
+    <section class="headline">    
+        <div class="nameTitle">
+            <h1 id="myName">Benjamin Semah</h1>
+            <h2 id="myTitle">Software Developer</h2>
+        </div>
+        <div id="bio">
+            <p id="shortBio">
+                I can help you build a product, feature or website.
+                Look through some of my work and experience! 
+                If you like what you see and have a project you need coded,  
+                don’t hestiate to contact me.
+            </p>
+        </div>
+    </section>
```

Comment:
- [ ] Your headline section is not well aligned to the centre of the page. I believe it's the image pushing it to the right a bit. That's why I suggested above, you try setting the image as a background image.

- [ ] Secondly, the images are not rendering as expected. Please take a look at your design to see the point am making here. Please do not forget to also check that you use the right fonts, font size and colour for the headline contents.

Your Design              |
:--------------------------:|
![Your design](https://user-images.githubusercontent.com/39523312/143461564-541be63e-1bdb-4a3e-9601-ed6558b9932b.png)